project.xml

```diff
@@ -15,6 +15,10 @@
     <use project = "systemd" optional = "1" implied = "1" min_major = "200" />
     <use project = "lz4" optional = "1" implied = "1" />
     <use project = "libcurl" optional = "1" implied = "1" min_major = "7" min_minor = "28" />
+    <use project = "libmicrohttpd"
+         tarball = "http://ftp.gnu.org/gnu/libmicrohttpd/libmicrohttpd-0.9.61.tar.gz"
+         implied = "1" optional = "1" min_major = "0" min_minor = "9" min_patch = "61"  />
```

Comment:
@somdoron any reason for this min version? It's what breaks using the package - it's not available anywhere yet

Reply:
Yes, but we might be able to reduce it, let me check, because I'm using some method that is only available recently

Reply:
Here you can find what's available on Ubuntu:

https://packages.ubuntu.com/search?suite=all&section=all&arch=any&keywords=libmicrohttpd-dev&searchon=names

Reply:
Ah ok that's even better - there's another change to do for backward compatibility:

```
--- a/src/zhttp_server.c
+++ b/src/zhttp_server.c
@@ -108,7 +108,11 @@ server_actor_new (zsock_t *pipe, zhttp_server_options_t *options)
     assert (self);
 
     self->daemon = MHD_start_daemon (
+#ifdef MHD_ALLOW_SUSPEND_RESUME
             MHD_ALLOW_SUSPEND_RESUME, // TODO: use epoll on linux
+#else
+            MHD_USE_SUSPEND_RESUME, // TODO: use epoll on linux
+#endif
             (uint16_t ) zhttp_server_options_port (options),
             NULL, NULL,
             (MHD_AccessHandlerCallback) s_handle_request, self,
```

Reply:
its an enum (:

Reply:
but they have the same value, so I will use the old one for now

Reply:
https://travis-ci.org/somdoron/czmq

Reply:
Ah, the upper case fooled me - it's easy to add an autoconf/cmake check if you want

Reply:
We can just have a version check, do you know which version it was introduced?

Reply:
0.9.53 https://github.com/Karlson2k/libmicrohttpd/commit/959d55ec5eea5b5d68fed524d1b48184c4763620

Reply:
yes, just found it as well.
Made the change, CI is building, will make the pull request.
I will be in the airplane real soon, so hopefully everything will pass.

Reply:
ok, safe travels!